### ISSUE TITLE
Make comma placement (L019) a fixable lint

### ISSUE THREAD

**Issue by pwildenhain** [sqlfluff/sqlfluff#453]
This _seems_ like it should be a pretty straightforward implementation -- place the commas before of after depending on the config?

Let me know if it's trickier than that -- if not, I'd be willing to take a stab at it. I think this would be a nice feature in a 0.4.0 release

Current behavior:

experiment.sql
```sql
select
    field,
    field1,
    field2
    , field3
from my_table
```

Lint ✔️ 
```sh
$ sqlfluff lint experiment.sql
== [experiment.sql] FAIL
L:   5 | P:   5 | L019 | Found leading comma. Expected only trailing.
```

Fix 🙅‍♂️ 
```sh
$ sqlfluff fix experiment.sql --rules L019
==== finding fixable violations ====
==== no fixable linting violations found ====
  [1 unfixable linting violations found]
```

**Comment by alanmcruickshank** [comment]
I think this should be doable yes. The key parts to make sure we deal with correctly are:
- trailing vs leading comma configuration.
- appropriate whitespace and indentation around the code that you edit.
- you may occasionally have to flow commas around comments, so it may be several lines later. I've included a test case below which I think should cover one of the tricky cases.

This query:
```
SELECT
    a.foo
    , a.bar,

    -- This is an important comment with awkward line spacing

   a.foobar,

    /* Which might also
    be followed by a multiline one */
    a.barfoo
from a
```

Should correct to this (if commas is set to leading):
```
SELECT
    a.foo
    , a.bar

    -- This is an important comment with awkward line spacing

   , a.foobar

    /* Which might also
    be followed by a multiline one */
    , a.barfoo
from a
```
